x-pack/libbeat/tests/system/test_management.py

```diff
@@ -25,10 +27,13 @@ def setUp(self):
         self.es_user = "myelastic"
         self.es_pass = "changeme"
         self.es = Elasticsearch([self.get_elasticsearch_url()], verify_certs=True)
+        self.keystore_path = self.working_dir + "/data/keystore"
```

Comment:
In general, better use `os.path.join`. We should also consider to 'abstract' some functionality around beats into classes.

Reply:
I will make that change, I can create an issue for abstract.